src/commands/links_execute.rs

```diff
@@ -0,0 +1,115 @@
+/// Module for determining execution target for resources with links
+use crate::commands::links_output::{ResourceLinks, ResourceType};
+
+#[derive(Debug, PartialEq)]
+pub enum ExecuteTarget {
+    Resource(String),
+    Label { label: String, app: String },
+}
```

Comment:
this might be clearer?

```suggestion
#[derive(Debug, PartialEq)]
pub enum ResourceTarget {
    ByName(String),
    ByLabel { label: String, app: String },
}
```